src/utilities/AlertHelper.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import Note from '../components/Note';
+import { SetFieldValues } from "../utilities/FormHelpers";
+export const getAlertMessage = (props) => {
+
+	const { hasPasswordReset } = props.values;
+	if (hasPasswordReset) {
+		return <Note className='bc_alert alert-success'>{props.values.SignInPage.ResetPasswordAlert.replace('{email address}', props.history.location.state)}</Note>
+	}
+	else if (props.status) 
```

Comment:
don't need this check here, if it's throwing an error, you can changed the const declartion below to

```js
const { incorrectEmail, networkError } = props.status || {};
```

this will ensure an error isn't thrown.

Reply:
updated